src/Game/Scenes/GameSceneInputHandler.cs

```diff
@@ -823,12 +957,13 @@ internal override void OnKeyUp(SDL.SDL_KeyboardEvent e)
 
             _useObjectHandles = isctrl && isshift;
 
-            if (e.keysym.sym == SDL.SDL_Keycode.SDLK_TAB && !Engine.Profile.Current.DisableTabBtn)
+            if (e.keysym.sym == SDL.SDL_Keycode.SDLK_TAB /*&& !Engine.Profile.Current.DisableTabBtn*/)
```

Comment:
Here reverted fix

Reply:
Fixed both instances as to not overwrite your recent commits.